packages/angular_devkit/build_angular/src/webpack/utils/stats.ts

```diff
@@ -21,11 +21,13 @@ export function formatSize(size: number): string {
 
   const abbreviations = ['bytes', 'kB', 'MB', 'GB'];
   const index = Math.floor(Math.log(size) / Math.log(1024));
-
-  return `${+(size / Math.pow(1024, index)).toPrecision(3)} ${abbreviations[index]}`;
+  const roundedSize = size / Math.pow(1024, index);
+  // bytes don't have a fraction
+  const fractionDigits = (roundedSize % 1) + index === 0 ? 0 : 2;
```

Comment:
The modulo seems to be a mistake, as it's always 0?

```suggestion
  const fractionDigits = index === 0 ? 0 : 2;
```

Reply:
The roundedSize in this case can contain decimal places, because the rounded is rounded to the nearest size.

Example
```ts
const size = 3000;
const roundedSize  = size / Math.pow(1024, 1); // 1 here is the index of KB
 // => 2.9296875

```

Reply:
Oh, that is trippy! But if `index === 0` then we must be dealing with bytes already, which are known to be integral?

Reply:
Ohh indeed